tmc-langs-util/src/test/java/fi/helsinki/cs/tmc/langs/util/TaskExecutorImplTest.java

```diff
@@ -0,0 +1,57 @@
+
+package fi.helsinki.cs.tmc.langs.util;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import fi.helsinki.cs.tmc.langs.domain.NoLanguagePluginFoundException;
+import fi.helsinki.cs.tmc.langs.java.ant.AntPlugin;
+import fi.helsinki.cs.tmc.langs.utils.TestUtils;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.LinkedList;
+import java.util.List;
+
+import org.apache.commons.compress.archivers.ArchiveException;
+import org.apache.commons.compress.archivers.ArchiveStreamFactory;
+import org.apache.commons.compress.archivers.tar.TarArchiveEntry;
+import org.apache.commons.compress.archivers.tar.TarArchiveInputStream;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.IOUtils;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+
+public class TaskExecutorImplTest {
+    
+    @Test
+    public void prepareSandboxTaskTest() throws IOException, NoLanguagePluginFoundException, ArchiveException {
+        TaskExecutor executor = new TaskExecutorImpl();
+
+        Path exercisePath = TestUtils.getPath(getClass(), "arith_funcs");
+        Path submissionPath = TestUtils.getPath(getClass(), "arith_funcs_solution");
+        Path outputDirectory = Files.createTempDirectory("output-directory");
+        Path outputPath = new File("/tmp/output.tar").toPath();//outputDirectory.resolve("output.tar");
+        Path tmcRunPath = new  File("/home/local/onniaarn/Visulahti.R").toPath();//Files.createTempFile("fake-tmc-run", ".sh");
+        Path tmcLangsPath = new  File("/home/local/onniaarn/Projects/tmc-langs/tmc-langs-cli/target/tmc-langs-cli-0.7.7-SNAPSHOT.jar").toPath();//Files.createTempFile("fake-tmc-langs", ".jar");
+
+        executor.prepareSandboxTask(exercisePath, submissionPath, outputPath, tmcRunPath, tmcLangsPath);
+        Path testDirectory = Files.createTempDirectory("test-directory");
+
+        Process untarringProcess = new ProcessBuilder("tar", "-C", testDirectory.toString(), "-xf", outputPath.toString()).start();
```

Comment:
The offending test is now set to be skipped if the functionality is not available, i.e. when on Windows. Shouldn't be an issue since the code will only ever be deployed to Linux. This solution is @nygrenh approved.